modules/tinymce/src/themes/silver/main/ts/ui/dialog/WindowManager.ts

```diff
@@ -181,11 +181,10 @@ const setup = (extras: WindowManagerSetup) => {
       inlineDialog.set(inlineDialogComp);
 
       // Position the inline dialog
-      InlineView.showWithin(
+      InlineView.showAt(
```

Comment:
This was incorrect, as it caused dialog to be placed incorrectly in inline mode since the body can be much smaller than the viewport. Additionally I'm pretty certain we actually want the dialogs to be constrained to the window viewport, so it made more sense to change it to this. I was actually the one who added this according to git, but I can't recall why 🤔